### PR TITLE
Tabelize linked output of `ls`

### PR DESCRIPTION
**Pull request by Master-Guy**
Undocumented bug:
The clickable files in `ls` were not put in table form, even though it was expected them to be.
The other files and folders were correctly formatted.

image

The new setup does correctly form "tables":

image

### CODE REVIEW

File: src/Terminal/commands/ls.tsx [hunk 1]
```diff
@@ -139,10 +139,12 @@ export function ls(
       }),
     )();
 
-    const rowSplit = row
-      .split(" ")
-      .map((x) => x.trim())
-      .filter((x) => !!x);
+    console.log(row);
```
Comment (ApamNapat) [inline src/Terminal/commands/ls.tsx:142]: Are this `console.log` and the one below supposed to be merged or were they for debugging?

File: src/Terminal/commands/ls.tsx [hunk 2]
```diff
@@ -139,10 +139,12 @@ export function ls(
...
+
+    const rowSplit = row.split("~");
+    let rowSplitArray = rowSplit.map((x) => [x.trim(), x.replace(x.trim(), "")]);
+    console.log(rowSplitArray);
+    rowSplitArray = rowSplitArray.filter((x) => !!x);
```
Comment (ApamNapat) [inline src/Terminal/commands/ls.tsx:147]: When would this filter anything? `rowSplitArray`'s elements are two element arrays, so they are always truthy, aren't they?

**Review by ApamNapat — CHANGES_REQUESTED** [review]

File: src/Terminal/commands/ls.tsx (hunk 1 above)
Comment (Master-Guy) [reply src/Terminal/commands/ls.tsx:142]: That should be removed, will do.

File: src/Terminal/commands/ls.tsx (hunk 2 above)
Comment (Master-Guy) [reply src/Terminal/commands/ls.tsx:147]: It was already in there, so I left it, but didn't change it to look at the first element of the array.

**Review by phyzical — APPROVED** [review]
lgtm

**Pull request merged**